projects/js-packages/publicize-components/src/components/post-publish-manual-sharing/index.jsx

```diff
@@ -29,6 +37,7 @@ export default function PostPublishManualSharing() {
 			<ThemeProvider>
 				<ManualSharingInfo className={ styles.description } variant="body-small" />
 				<ShareButtons />
+				{ featureFlags?.useShareStatus && 'Share status modal comes here' }
```

Comment:
This component is about manual sharing but we added share status here. I think it will be good to have this as its own component.

IMHO, we should
- Move `PostPublishPanels` component to `publicize-components`
- Use that to display all our post publish panels
- Use that both in Jetpack and Social to avoid updating all the panels in both the plugins.

Reply:
definitely, I just added this part for the sake of testing but that refactor sounds reasonable. Maybe another task that can be done before going forward with the project tasks?

Reply:
Yes, it is better to avoid a larger refactor later.